### ISSUE TITLE
Is there a way to increase read timeout for article download?

### ISSUE THREAD

**Issue by 5465869** [codelucas/newspaper#871]
Hi, I am attempting to use newspaper to download many articles and do not want the timeout window to be set at 7 seconds. Is there any way either within the package or externally to avoid the timeout exception when downloading?

**Comment by johnbumgarner** [comment]
Yes, the timeout is configurable.   Take a look at the examples in my [newspaper3 usage overview document](https://github.com/johnbumgarner/newspaper3_usage_overview).
